src/main/java/com/microfocus/octane/plugins/configuration/OctaneRestManager.java

```diff
@@ -88,7 +88,7 @@ private static String getQueryParamBasedOnTypeAndVersion(SpaceConfiguration sc,
         return  getQueryParam(octaneEntity, typeDescriptor);
     }
 
-    private static VersionEntity getOctaneServerVersion(String spaceConfigId) {
+    public static VersionEntity getOctaneServerVersion(String spaceConfigId) {
         SpaceConfiguration sc = ConfigurationManager.getInstance().getSpaceConfigurationById(spaceConfigId, true).get();
```

Comment:
Since in this method you get the space configuration again by id, maybe you can pass the space config directly (you have it in your case when calling the method) - see if you can refactor it for the other case too